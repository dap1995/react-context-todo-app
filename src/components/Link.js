import styled from 'styled-components';

const Link = styled.a`
  display: flex;
  flex: 1;
  align-items: center;
  justify-content: center;
  color: #bfbfbf;
  background-color: white;
  padding: 5px;
  margin: 5px;
  font-size: 12px;
  font-weight: 300;
`;

export default Link;
