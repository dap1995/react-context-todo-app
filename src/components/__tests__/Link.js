import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Link from '../Link';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Link onClick={() => {}} />, div);
});

it('should be the same as the snapshot', () => {
  const tree = renderer.create(<Link />).toJSON();
  expect(tree).toMatchSnapshot();
});
