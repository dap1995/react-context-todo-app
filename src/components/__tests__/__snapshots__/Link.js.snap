// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be the same as the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  color: #bfbfbf;
  background-color: white;
  padding: 5px;
  margin: 5px;
  font-size: 12px;
  font-weight: 300;
}

<a
  className="c0"
/>
`;
